src/Migrator.php

```diff
@@ -226,4 +245,102 @@ protected function migrationTable(string $database = null): Table
     {
         return $this->dbal->database($database)->table($this->config->getTable());
     }
+
+    protected function checkMigrationTableStructure(Database $db): bool
+    {
+        $table = $db->table($this->config->getTable());
+
+        foreach (self::MIGRATION_TABLE_FIELDS_LIST as $field) {
+            if (!$table->hasColumn($field)) {
+                return false;
+            }
+        }
+
+        if (!$table->hasIndex(['migration', 'created_at'])) {
```

Comment:
For ideal case it hsould check if index is unique but I suppose it is not required...

Reply:
Not required.